### PR TITLE
Added Code of Conduct and linked it in README.md

### LINKED ISSUE #54: Code_Of_Conduct.md not there in the repository
What documentation needs to be updated?
The Code of Conduct documentation for the twinTrim repository.

Describe the changes needed -:

> Add a more detailed Enforcement Guidelines section that clearly outlines how violations of the Code of Conduct will be reported, investigated, and resolved.

> Remove any specific maintainer email in the Reporting Violations section to make it generic for all contributors.

> Ensure that the documentation includes sections on consequences for unacceptable behavior, the right to appeal, confidentiality, and protection from retaliation.

Additional context - :

> The Code of Conduct should follow a recognized standard such as the Contributor Covenant, but be customized for the twinTrim project.

> The goal is to provide clarity on how reports of misconduct will be handled, ensuring contributors understand the process and their rights within the community.
Please assign me this issue so I can add it. Please add gssoc ext and hacktoberfest labels

### PR DESCRIPTION
**Pull request by ShirshenduR**
## Description

This pull request adds a **Code of Conduct** file to the repository to ensure contributors are aware of and adhere to community standards. It includes detailed **Enforcement Guidelines** outlining the procedure for reporting violations, handling investigations, and outlining potential consequences. The **Code of Conduct** has been linked in the `README.md` for easy access.

- Fixes #54 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Tests update

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas (if applicable).
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have maintained a clean commit history by using the necessary Git commands.
- [x] I have checked that my code does not cause any merge conflicts.

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Comment by Kota-Karthik** [comment]
@ShirshenduR 
Very nice of you copying it from BotVerse 
you should have done minimum amount of work to change its name from `BotVerse` to `TwinTrim`
make necessary changes

**Comment by ShirshenduR** [comment]
> @ShirshenduR Very nice of you copying it from BotVerse you should have done minimum amount of work to change its name from `BotVerse` to `TwinTrim` make necessary changes

made the necessary changes :)

**Comment by ShirshenduR** [comment]
> @ShirshenduR DO NOT, and I repeat it, do not just copy paste things otherwise your contributions will not be considered.

ok i will not do that

**Comment by techy4shri** [comment]
@ShirshenduR that is not how things work. First rule of opensource contribution here, you do not copy paste things, read what you have written. I do not care for what you did in other repositories, here, you follow rules and make meaningful contributions. So change the Code Of Conduct, otheriwse this PR will be rejected.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+The following actions will not be tolerated:
+
+- Any form of harassment, including derogatory comments, insults, or threats.
+- Use of sexualized language or unwelcome advances.
+- Personal attacks, trolling, or political arguments that derail discussions.
+- Publishing or sharing private information of others without consent.
+- Any behavior that creates a hostile or unsafe environment.
+
+## Responsibility of Maintainers
```
Comment (Wahid7852) [inline CODE_OF_CONDUCT.md:27]: if its a code of conduct for contributors (line #1), why add this point here

**Pull request closed without merging**